Automattic/vip-go-mu-plugins
# Update ElasticPress to latest

## Description

Updates the ElasticPress submodule to the latest on Automattic/ElasticPress#develop.

## Changelog Description

### Search: Update ElasticPress

This updates the ElasticPress submodule to [da9e58e](https://github.com/Automattic/ElasticPress/commit/da9e58e74d08d2e9c4ff5ecdec43cb8144bc019e), which adds utility functions for getting and updating an index's settings.

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).
- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

1. Pull down PR
2. Update submodules - `git submodule update --init`
2. Perform a variety of searches and verify that ES requests/responses are working as expected
3. Get an indexable's settings - in `wp shell` - `\ElasticPress\Indexables::factory()->get( 'post' )->get_index_settings()`
4. The above should print out the current settings for the post index
